stages/org.osbuild.users

```diff
@@ -48,8 +49,7 @@ def useradd(
     if expiredate is not None:
         arguments += ["--expiredate", str(expiredate)]
 
-    with Chroot(root) as chroot:
-        chroot.run(["useradd", *arguments, name], check=True)
+    subprocess.run(["useradd", "--root", root, *arguments, name], check=True)
```

Comment:
I think that this change and the others will also require modifying the unit tests:
https://github.com/osbuild/osbuild/blob/8e3d0540997770067238d19e7dc8216c209fb4da/stages/test/test_users.py#L61-L65

Reply:
Oh yes, of course!  I was focused on the stage test and didn't bother with the rest.

Thanks!